airflow/providers/amazon/aws/transfers/gcs_to_s3.py

```diff
@@ -88,6 +89,7 @@ class GCSToS3Operator(BaseOperator):
 
     template_fields: Sequence[str] = (
         "bucket",
+        "gcs_bucket",
```

Comment:
now that we have `gcs_bucket` we can safely remove `bucket` from the list dont we?

Reply:
Yes. I see now the "rendering" of templated fields occurs on `execute()` so no need to include the deprecated param here. Removed in 03c4bc8. Thanks.